PolicyNetPlugin/src/org/workcraft/plugins/policy/PolicyNetToolProvider.java

```diff
@@ -18,7 +18,7 @@
     public Iterable<GraphEditorTool> getTools() {
         ArrayList<GraphEditorTool> result = new ArrayList<>();
 
-        result.add(new SelectionTool(false));
```

Comment:
It looks wrong -- if false is not passed, then creation of pages will be enabled in this model.